.eslintrc.cjs

```diff
@@ -43,6 +43,10 @@ module.exports = {
 			style: { lang: 'scss' }
 		} ],
 
-		'@typescript-eslint/type-annotation-spacing': 'error'
+		'@typescript-eslint/type-annotation-spacing': 'error',
+
+		// ESLint treats Typescript enums weird
```

Comment:
```suggestion
		// Workaround for Typescript enums, see https://github.com/typescript-eslint/typescript-eslint/issues/2483
```
When I saw this I had no context, was baffled and needed to search for "eslint typescript enums"